java-maven/README.md

```diff
@@ -10,17 +10,18 @@ This sample may be out of date. If you face any issues, please reach out to mail
 * [Using docker compose](#using-docker-compose)
 * [Rookout Integration explained](#rookout-integration-explained)
 ## Running locally
-1. Run the Rookout agent:
-    ``` bash
-    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-    ```
-2. Compile the project jar and download the Java agent:
+1. Clone and compile the project jar and download the Java agent:
```

Comment:
@ManuelNemirovsky is this on the same PR intentionally?

Reply:
No, 
I got confused between the branches when updating the java-maven readme file, but its supposed to be the same code.

Reply:
@ManuelNemirovsky and this file?